src/Api.php

```diff
@@ -180,7 +180,13 @@ private static function getProductsImageUrl($product)
             return '';
         }
 
-        return (new Link())->getImageLink($link_rewrite, $image['id_image']);
+        $imageUrl = (new Link())->getImageLink($link_rewrite, $image['id_image']);
+
+        if (strpos($imageUrl, 'http') === false) {
```

Comment:
my gdzieś z API http dajemy, czy to w preście?

Reply:
To z Presty. Okazało się, że nie dokleja protokołu i u nas na wniosku nie wyświetlają się zdjęcia produktów z koszyka przekazanego do nas.